shim/linux/src/entry.c

```diff
@@ -458,8 +553,11 @@ dispatch_vm_kvm_xen_hvm_config(struct kvm_xen_hvm_config *const ioctl_args)
 
 static long
 dev_unlocked_ioctl_vm(
-    struct file *file, unsigned int cmd, unsigned long ioctl_args)
+    struct file *filep, unsigned int cmd, unsigned long ioctl_args)
 {
+
```

Comment:
Remove the extra line here. Also, I would add a platfrom_expects(nullptr != VM) after getting private data just to be on the safe side.